README.rst

```diff
@@ -8,6 +8,13 @@
 Qiskit on IQM
 #############
 
+**The** ``qiskit-iqm`` **package is deprecated and the GitHub repository has been archived. New versions of
+Qiskit on IQM will be published as part of the** `iqm-client <https://pypi.org/project/iqm-client/>`_
+**package and can be installed from public PyPI with** ``pip install iqm-client[qiskit]``.
+**See the latest documentation at** `<https://docs.meetiqm.com/iqm-client/user_guide_qiskit.html>`_ **for more
```

Comment:
Note that this link to `user_guide_qiskit.html` will not work until the new `iqm-client` version containing `qiskit_iqm` is published.